code/nodes/runModels/funcs.q

```diff
@@ -0,0 +1,143 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of runModels
+
+// @kind function
+// @category runModels
+// @fileoverview Set value of random seed for reproducability
+// @param cfg {dict} configuration information relating to the current run of AutoML
+// @return {Null} Value of seed is set
+runModels.setSeed:{[cfg]
+  system"S ",string cfg`seed;
+  }
+
+// @kind function
+// @category runModels
+// @fileoverview Apply train test split to keep holdout for feature impact plot and testing of vanilla best model
+// @param cfg  {dict} Configuration information relating to the current run of AutoML
+// @param tts  {dict} Feature and target data split into training and testing set
+// @return {dict} Training and holdout split of data
+runModels.holdoutSplit:{[cfg;tts]
+  ttsFunc:get cfg`tts;
```

Comment:
This will need to be able to take the python equivalent too in line with the updated train-test split functionality. I'll merge this node after Deanna's tts/significance node so this should be updated in line